### ISSUE TITLE
Font's Lowercase Characters shifting upward automatically!

### ISSUE THREAD

**Issue by anandrajgupta** [hzeller/rpi-rgb-led-matrix#1426]
Hi, Thank you Mr. Henner Zeller...

I am using otf2bdf font creator to make various sized fonts, The issue I am facing is that when we write Uppercase characters then everything is fine but when writing Lowercase characters they are shifted upward automatically, their baseline kind of shifting upward which makes them look weird as shown in below picture:

![fontLowerCaseIssue](https://user-images.githubusercontent.com/19954301/164170493-c238a8e1-67cd-40ba-910e-3cbb2c8d1c7f.jpg)

Note: The same issue is not observed when using one of the fonts provided with this library e.g. 8x13B.bdf

Any help will be highly appreciated!

**Comment by hzeller** [comment]
Do you have the BDF file for me ? I think it might happen if the font decides to choose a different baseline for each glyph.

**Comment by anandrajgupta** [comment]
Yes sir, here it is:
[testFont.zip](https://github.com/hzeller/rpi-rgb-led-matrix/files/8519224/testFont.zip)

**Comment by anandrajgupta** [comment]
Sir how to configure/modify the font in order to make it use same baseline for each glyph?

Thank you...

**Comment by hzeller** [comment]
So the text `Ready` looks ok for me on my screen.

**Comment by anandrajgupta** [comment]
The issue has been solved sir!
I had to tick the checkbox saying "don't trim glyph bitmaps" at the time of exporting font from the windows tool I'm using i.e. "Fony", as shown in below screenshot:

![Screenshot (48)](https://user-images.githubusercontent.com/19954301/164176087-347fc118-4d77-4544-86b8-e438f75eca79.png)

Now, I just have an additional step involved after making any new font using otf2bdf, i.e. feed the otf2bdf generated font into this "Fony" tool then export by unchecking that trim checkbox and the font looks as expected!

In your case you are saying "Ready" looks ok on your screen then maybe there is some other mystery behind it! while I am also using latest updated library. Anyways, I have found a way to overcome the problem I was facing so I'm happy!

Once again, Thank you very very much for your kind support!

Closing this issue now!

**Comment by anandrajgupta** [comment]
The issue has been solved sir!
I had to tick the checkbox saying "don't trim glyph bitmaps" at the time of exporting font from the windows tool I'm using i.e. "Fony", as shown in below screenshot:

![Screenshot (48)](https://user-images.githubusercontent.com/19954301/164176087-347fc118-4d77-4544-86b8-e438f75eca79.png)

Now, I just have an additional step involved after making any new font using otf2bdf, i.e. feed the otf2bdf generated font into this "Fony" tool then export by unchecking that trim checkbox and the font looks as expected!

In your case you are saying "Ready" looks ok on your screen then maybe there is some other mystery behind it! while I am also using latest updated library. Anyways, I have found a way to overcome the problem I was facing so I'm happy!

Once again, Thank you very very much for your kind support!

Closing this issue now!
